packages/manager/modules/vps/src/dashboard/tile/configuration/index.html

```diff
@@ -11,6 +11,14 @@
     <oui-tile-definition
         data-term="{{ ::'vps_dashboard_tile_configuration_vcore' | translate }}"
     >
+        <oui-tile-description data-ng-if="$ctrl.statistics">
+            <progress
+                class="oui-progress"
+                data-ng-class="'oui-progress_' + ($ctrl.statistics.cpu <= RTM_WARNING_VALUE.min ? 'success' : $ctrl.statistics.cpu <= RTM_WARNING_VALUE.max ? 'warning' : 'error')"
```

Comment:
```suggestion
                data-ng-class="{ 
                  'oui-progress_success': $ctrl.statistics.cpu <= RTM_WARNING_VALUE.min,
                  'oui-progress_warning' : $ctrl.statistics.cpu > RTM_WARNING_VALUE.min && $ctrl.statistics.cpu <= RTM_WARNING_VALUE.max,
                  'oui-progress_error': $ctrl.statistics.cpu > RTM_WARNING_VALUE.max,
                }"
                  
```

or to avoid comparison in template :

```suggestion
                data-ng-class="{ 
                  'oui-progress_success': $ctrl.isCpuUsageNormal(),
                  'oui-progress_warning' : $ctrl.isCpuUsageWarning(),
                  'oui-progress_error': $ctrl.isCpuUsageHigh(),
                }"
                  
```